From foo@baz Tue Apr  9 12:12:43 2002
To: Greg KH <greg@kroah.com>
Date: 02 Mar 2005 12:10:01 -08:00
From: mhoffman@lightlink.com
Subject: [PATCH] I2C: unnecessary #includes in asb100.c

* Jean Delvare <khali@linux-fr.org> [2005-01-25 10:14:49 +0100]:
> Any reson why asb100.c (in linux 2.6.11-rc2) includes linux/ioport.h and
> asm/io.h? As an i2c-only chip driver, I don't think it needs these.
>
> As a side note, I also wonder what the inclusions of linux/config.h,
> linux/types.h and asm/errno.h are there for.

Because they look pretty?  Here's a patch Greg, please apply...

Signed-off-by: Mark M. Hoffman <mhoffman@lightlink.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


diff -Nru a/drivers/i2c/chips/asb100.c b/drivers/i2c/chips/asb100.c
--- a/drivers/i2c/chips/asb100.c	2005-03-02 16:33:26 -08:00
+++ b/drivers/i2c/chips/asb100.c	2005-03-02 16:33:26 -08:00
@@ -36,17 +36,12 @@
     asb100	7	3	1	4	0x31	0x0694	yes	no
 */
 
-#include <linux/config.h>
 #include <linux/module.h>
 #include <linux/slab.h>
-#include <linux/ioport.h>
-#include <linux/types.h>
 #include <linux/i2c.h>
 #include <linux/i2c-sensor.h>
 #include <linux/i2c-vid.h>
 #include <linux/init.h>
-#include <asm/errno.h>
-#include <asm/io.h>
 #include "lm75.h"
 
 /*
